gunicorn/workers/gtornado.py

```diff
@@ -105,9 +105,12 @@ def run(self):
         # instance of tornado.web.Application or is an
         # instance of tornado.wsgi.WSGIApplication
         app = self.wsgi
-        if not isinstance(app, tornado.web.Application) or \
-           isinstance(app, tornado.wsgi.WSGIApplication):
-            app = WSGIContainer(app)
+
+        # Patch for tornado5
+        if tornado.version_info[0] == 5:
```

Comment:
The pull request description says Tornado 6, but this seems to show Tornado 5. Is this for 5 and 6?

Reply:
This check (conversion) is only required on `tornado` **5**, this is not required on `tornado` **6**
@see https://www.tornadoweb.org/en/stable/releases/v6.0.0.html#tornado-wsgi

BTW, in deed this is **breaking** for `tornado` **<5**, but I honestly do not think it's a big deal

Reply:
Can you explain what you mean by this breaking for Tornado < 5? You mean that this block should only be run for Tornado 5 (and not 4 or 6)? Or do you mean that having this block at all, even with the `version_info` guard, breaks Gunicorn for Tornado 4 somehow?

Reply:
Oh, I see, this was not in the guard before.

Can you change this to be < 6 then, and we can merge it? Otherwise, I can merge it but I can't backport it to the 19.x branch.